packages/cli/src/runners/runner-ws-server.ts

```diff
@@ -21,15 +21,17 @@ function heartbeat(this: WebSocket) {
 	this.isAlive = true;
 }
 
-const enum WsStatusCode {
-	CloseNormal = 1000,
-	CloseGoingAway = 1001,
-	CloseProtocolError = 1002,
-	CloseUnsupportedData = 1003,
-	CloseNoStatus = 1005,
-	CloseAbnormal = 1006,
-	CloseInvalidData = 1007,
-}
+const WsStatusCodes = {
```

Comment:
can we move these to `constants.ts`? that way we can update [push code](https://github.com/n8n-io/n8n/blob/master/packages/cli/src/push/index.ts#L66) to use these as well.